app/models/vm_or_template/operations.rb

```diff
@@ -118,6 +118,16 @@ def validate_vm_control
       msg ||= _("Provider doesn't support vm_destroy") unless ext_management_system.respond_to?(:vm_destroy)
       unsupported_reason_add(:terminate, msg) if msg
     end
+
+    supports :suspend do
+      msg = unsupported_reason(:control) unless supports_control?
+      msg ||= _("The VM is not powered on") unless vm_powered_on?
+      unsupported_reason_add(:suspend, msg) if msg
+    end
+  end
+
+  def vm_powered_on?
+    current_state == "on"
```

Comment:
I can fix instances_spec.rb test by change
let(:ems) { FactoryGirl.create(:ems_openstack_infra, :zone => zone) }
to
let(:ems) { FactoryGirl.create(:ems_openstack, :zone => zone) }
but I do not sure if it desired behavior